Pocket Casts Watch App Extension/PodcastEpisodeListView.swift

```diff
@@ -21,19 +24,26 @@ struct PodcastEpisodeListView: View {
     }
 
     var podcastInfo: some View {
-        Group {
-            CachedImage(url: viewModel.podcast.artworkURL)
-                .frame(width: headerImageSize, height: headerImageSize, alignment: .center)
-            Text(viewModel.podcast.title ?? "")
-            Text(viewModel.podcast.author ?? "")
-                .font(.dynamic(size: 12))
-                .foregroundColor(.subheadlineText)
-                .padding(.bottom, 3)
-            if viewModel.episodes.isEmpty {
-                Text(L10n.watchNoEpisodes)
-                    .font(.subheadline)
+        HStack {
+            Spacer()
+            VStack(alignment: .center) {
+                CachedImage(url: viewModel.podcast.artworkURL)
+                    .frame(width: headerImageSize, height: headerImageSize, alignment: .center)
+                Text(viewModel.podcast.title ?? "")
+                    .font(.dynamic(size: 16))
+                    .foregroundColor(.white)
+                Text(viewModel.podcast.author ?? "")
+                    .font(.dynamic(size: 12))
+                    .foregroundColor(.subheadlineText)
+                    .padding(.bottom, 3)
+                if viewModel.episodes.isEmpty {
+                    Text(L10n.watchNoEpisodes)
+                        .font(.subheadline)
+                }
             }
+            Spacer()
         }
+        .padding(.horizontal, -8)
```

Comment:
We can horizontally center the VStack by using `.frame(maxWidth: .infinity)` which forces it to take up the entire horizontal space. 

```suggestion
        VStack(alignment: .center) {
            CachedImage(url: viewModel.podcast.artworkURL)
                .frame(width: headerImageSize, height: headerImageSize, alignment: .center)
            Text(viewModel.podcast.title ?? "")
                .font(.dynamic(size: 16))
                .foregroundColor(.white)
            Text(viewModel.podcast.author ?? "")
                .font(.dynamic(size: 12))
                .foregroundColor(.subheadlineText)
                .padding(.bottom, 3)
            if viewModel.episodes.isEmpty {
                Text(L10n.watchNoEpisodes)
                    .font(.subheadline)
            }
        }
        .frame(maxWidth: .infinity)
        .multilineTextAlignment(.center)
```